1/test/run

```diff
@@ -137,7 +137,40 @@ function run_tests() {
   echo "  Success!"
 }
 
+function run_s2i_tests() {
+  if [ "$(which s2i)" == "" ]; then
+    echo "WARNING: Please install the s2i binary to run the s2i tests."
+    return
+  fi
+  output_image="openshift/s2i-test-jenkins"
+  s2i build --context-dir="1/test/s2i" --loglevel=3 --pull-policy=never \
+    file://$(pwd)/.. $IMAGE_NAME $output_image
+  IMAGE_NAME="${output_image}" create_container s2i-jenkins-test $VOLUME
+  cid=s2i-jenkins-test
+  sleep 3 # Give jenkins time to copy all plugins/configs
```

Comment:
it can flake then... the first case can succeed but it will fail for the second one (the files are not yet copied)... unless you want to retry the whole test :-) i don't think 3 seconds are all that horrible.